docs/evm/cadence/batched-evm-transactions.md

```diff
@@ -0,0 +1,641 @@
+Cadence Vault. Finally, it wraps FLOW as WFLOW, approves the ERC721 contract to move the mint amount, and attempts to
+mint the ERC721 token.
+
+Let's see what each step looks like in the transaction code.
+
+### COA Configuration
+
+The first step in our transaction is to configure a COA in the signer's account if one doesn't already exist. This is
+done by creating a new COA resource and saving it to the signer account's storage. A public Capability on the COA is
+then issued and published on the signer's account, allowing anyone to deposit FLOW into the COA, affecting its EVM
+balance.
+
+```cadence
+/* COA configuration & assigment */
+//
+let storagePath = /storage/evm
+let publicPath = /public/evm
+// Configure a COA if one is not found in storage at the default path
+if signer.storage.type(at: storagePath) == nil {
+    // Create & save the CadenceOwnedAccount (COA) Resource
+    let newCOA <- EVM.createCadenceOwnedAccount()
+    signer.storage.save(<-newCOA, to: storagePath)
+
+    // Unpublish any existing Capability at the public path if it exists
+    signer.capabilities.unpublish(publicPath)
+    // Issue & publish the public, unentitled COA Capability
+    let coaCapability = signer.capabilities.storage.issue<&EVM.CadenceOwnedAccount>(storagePath)
+    signer.capabilities.publish(coaCapability, at: publicPath)
+}
+
+// Assign the COA reference to the transaction's coa field
+self.coa = signer.storage.borrow<auth(EVM.Call) &EVM.CadenceOwnedAccount>(from: storagePath)
+    ?? panic("A CadenceOwnedAccount (COA) Resource could not be found at path ".concat(storagePath.toString())
+        .concat(" - ensure the COA Resource is created and saved at this path to enable EVM interactions"))
+```
+
+At the end of this section, the transaction now has an reference authorized with the `EVM.Call`
+[entitlement](https://cadence-lang.org/docs/language/access-control#entitlements) to use in the `execute` block which
+can be used call into EVM.
+
+### Funding the COA
+
+Next, we fund the COA with enough FLOW to cover the mint cost. This is done by withdrawing FLOW from the signer's
+FlowToken Vault and depositing it into the COA.
+
+```cadence
+/* Fund COA with cost of mint */
+//
+// Borrow authorized reference to signer's FlowToken Vault
+let sourceVault = signer.storage.borrow<auth(FungibleToken.Withdraw) &FlowToken.Vault>(
+        from: /storage/flowTokenVault
+    ) ?? panic("The signer does not store a FlowToken Vault object at the path "
+            .concat("/storage/flowTokenVault. ")
+            .concat("The signer must initialize their account with this vault first!"))
+// Withdraw from the signer's FlowToken Vault
+self.mintCost = 1.0
+let fundingVault <- sourceVault.withdraw(amount: self.mintCost) as! @FlowToken.Vault
+// Deposit the mint cost into the COA
+self.coa.deposit(from: <-fundingVault)
+```
+
+Taking a look at the full transaction, we can see an explicit check that the COA has enough FLOW to cover the mint cost
```

Comment:
I didn't have enough Flow and failed before this.  Got the error that there wasn't enough in the vault to transfer to.

Tried to repro by switching accounts and even after refreshing, I'm getting:  Error
 1 error occurred:
	* transaction verification failed: [Error Code: 1006] invalid proposal key: public key 0 on account d83e39fd1baa9ac3 does not have a valid signature: [Error Code: 1008] invalid payload signature: public key 0 on account d83e39fd1baa9ac3 does not have a valid signature: signature is not valid

Reply:
Looks like that account only has `0.001` FLOW, try again after funding from faucet. The other invalid signer errors may be wallet issues.